meeting-notes/2020/Q1/2020-01-27--core-implementations-weekly.md

```diff
@@ -0,0 +1,312 @@
+  - The remaining problems look like they may be related to the test but need more investigation.
+- Work on testing multiple versions underway.
+  - https://github.com/ipfs/testground/pull/400
+- Work on persisting assets under way
+  - https://github.com/ipfs/testground/pull/409
+
+#### Content Routing
+@stebalien / @raulk / @dirkmc / @aschmahmann / @jacobheun
+
+- All high level (Epic / Feature set) issues have been created
+- We will finish breaking down the work and estimating this week
+  - We will be finalizing the timeline based on these
+  - Issues will be created for each of the sub tasks if they don't already exist and associated to their parent issues.
+- Content routing plan has seen many updates: https://docs.google.com/document/d/1obd3QHL5QLAUHqi8oxtOl0EYs9bAuPVwz5SWQ53rSvI/edit#
+- Kademlia query logic PR has tests passing
+  - We need to run this against Testground
+- We have noticed spliting of the network (disjointed networks)  
+  - Need to work out a plan on how to test and resolve this issue
+  
+
+#### Subdomain Gateway (Base32, Origin isolation)
+@lidel
+
+- go-ipfs: [subdomain gateway feature](https://github.com/ipfs/go-ipfs/pull/6096) mostly implemented, remaining work: config cleanup, tests, docs and bugfixes
+  - A version will be implemented for js-ipfs in the near future
+
+#### Bitswap Updates
+@dirkmc
+- Running test suites against bitswap master vs [proof-of-concept](https://github.com/ipfs/go-bitswap/pull/189)
+  - Spoiler: More seeds makes things speed up now!
+
+#### js-ipfs/js-libp2p/js-ipld async await refactor
+@achingbrain / @jacobheun / @vasco-santos / @alanshaw
+
+- js-libp2p
+  - Releasing tomorrow (v0.27), January 28th! 🚀
+  
+- js-ipfs
+  - async/await PRs merged
+
+- Lets remove this initiative! \o/
+
+#### Stream Content-Based-Chunking Research/Improvements (go-ipfs / js-ipfs)
+@ribasushi
+
+- Failed to get a preliminary version of the chunker-tester published/demoed today :( Ironing out the last few off-by-ones / mismatches with go-ipfs. *REALLY* should have it all done by tomorrow morning.
+  - It's a cli tool that reads from stdin, the work hasn't been pushed yet. Will be pushed later today.
+
+### Other Initiatives
+
+This is the backlog of initiatives that are either on-hold or low-priority.
+
+#### UnixFS v1.5 (go-ipfs / js-ipfs)
+@achingbrain / @ribasushi
+
+#### Add Performance (go-ipfs / js-ipfs)
+@aschmahmann / @achingbrain
+
+#### Distributed Signaling (js-libp2p)
+@jacobheun
+
+- Flushing out the js-libp2p roadmap for this year and will assess the priority of this against other initiatives. After initial discussion this still seems to be high on the priority list.
+
+#### Migration to multihash keys in blockstore (js-ipfs / go-ipfs)
+@alanshaw / @auhau
+
+- from @stebalien to @hsanjuan: will you likely have time to tackle this in the near future?
+  - https://github.com/ipfs/go-ipfs/issues/6815
+
+### Design Review Proposals
```

Comment:
@achingbrain could you post a time/date here when you've found a time that works for the design review.